regolith/schemas.py

```diff
@@ -338,23 +359,40 @@
                                    '-PxiboYdM/edit?usp=sharing',
                        'year': 2017}],
                'title': 'Dr.'},
-    'presentations': {'_id': '18sb_ignobel',
-                      'abstract': 'We pulled apart graphite with tape',
-                      'authors': ['sbanerjee', 'cliu', 'sbillinge'],
-                      'begin_year': 2018,
-                      'begin_month': 'May',
-                      'begin_day': 22,
-                      'department': 'APAM',
-                      'end_year': 2018,
-                      'end_month': 'May',
-                      'end_day': 22,
-                      'institution': 'bnl',
-                      'meeting_name': '2018 NSLS-II and CFN Users Meeting',
-                      'project': '18sob_clustermining',
-                      'title': 'ClusterMining: extracting core structures of '
-                               'metallic nanoparticles from the atomic pair '
-                               'distribution function',
-                      'type': 'poster'},
+    'presentations': [{'_id': '18sb_ignobel',
```

Comment:
Could you limit the authors in the presentations to people who are in the people exemplar? The testing db is built off the exemplars and thus only can reference other exemplars.

Reply:
ok, I see.